specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -0,0 +1,2279 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Rules Engine Configuration": {
+            "$ref": "./examples/FrontdoorRulesEngineDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "FrontDoor": {
+      "properties": {
+        "properties": {
+          "description": "Properties of the Front Door Load Balancer",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/FrontDoorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "Front Door represents a collection of backend endpoints to route traffic to along with rules that specify how traffic is sent there."
+    },
+    "FrontDoorUpdateParameters": {
+      "description": "The properties needed to update a Front Door",
+      "properties": {
+        "friendlyName": {
+          "description": "A friendly name for the frontDoor",
+          "type": "string"
+        },
+        "routingRules": {
+          "description": "Routing rules associated with this Front Door.",
+          "type": "array",
+          "items": {
+            "description": "A routing rule",
+            "$ref": "#/definitions/RoutingRule"
+          }
+        },
+        "loadBalancingSettings": {
+          "description": "Load balancing settings associated with this Front Door instance.",
+          "type": "array",
+          "items": {
+            "description": "A load balancing settings",
+            "$ref": "#/definitions/LoadBalancingSettingsModel"
+          }
+        },
+        "healthProbeSettings": {
+          "description": "Health probe settings associated with this Front Door instance.",
+          "type": "array",
+          "items": {
+            "description": "A health probe settings",
+            "$ref": "#/definitions/HealthProbeSettingsModel"
+          }
+        },
+        "backendPools": {
+          "description": "Backend pools available to routing rules.",
+          "type": "array",
+          "items": {
+            "description": "A backend pool",
+            "$ref": "#/definitions/BackendPool"
+          }
+        },
+        "frontendEndpoints": {
+          "description": "Frontend endpoints available to routing rules.",
+          "type": "array",
+          "items": {
+            "description": "A frontend endpoint",
+            "$ref": "#/definitions/FrontendEndpoint"
+          }
+        },
+        "rulesEngines":
+        {
+          "description": "Rules Engine Configurations available to routing rules.",
+          "type": "array",
+          "items": {
+            "description": "A rules engine configuration.",
+            "$ref": "#/definitions/RulesEngine"
+          }
+        },
```

Comment:
having this here would mean all the rulesEngines can be created on PUT Frontdoor and also needs to be listed on GET Frontdoor. I think we don't need this since we are only reading and updating rules engine rules through subresources

Reply:
Removed rulesEngines.

---
In reply to: [365451391](https://github.com/Azure/azure-rest-api-specs/pull/8115#discussion_r365451391) [](ancestors = 365451391)